tests/test_modeling_flax_utils.py

```diff
@@ -265,7 +264,6 @@ def test_safetensors_load_from_hub_from_safetensors_pt(self):
         safetensors_model = FlaxBertModel.from_pretrained("hf-internal-testing/tiny-bert-pt-safetensors")
         self.assertTrue(check_models_equal(flax_model, safetensors_model))
 
-    @require_torch
```

Comment:
Removing these requirements is equivalent to testing the two codepaths that were tested by the two tests that are removed below